src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -9,7 +9,19 @@
       <v-card flat class="mt-5">
         <NameRequestInfo />
 
-        <!-- FUTURE: Business Type goes here -->
+        <template v-if="isTypePartnership">
+          <v-divider class="mx-6" />
+
+          <!-- Business Type -->
+          <header id="business-type-confirm-header" />
+          <BusinessTypeConfirm
+            class="py-8 px-6"
+            :class="{ 'invalid-section': getShowErrors && !businessTypeConfirmValid }"
+            :businessTypeConfirm="getRegistration.businessTypeConfirm"
+            @update:businessTypeConfirm="setRegistrationBusinessTypeConfirm($event)"
+            @valid="onBusinessTypeConfirmValidEvent($event)"
```

Comment:
You may need to pass the global error flag into your component to change is label red on error. Check with Yui what he wants.

Reply:
Okay. I'm on it.

Reply:
I see that you're looking at the conditional 'invalid-section' classname on the object to set the label text colour. Sneaky but if it works then 👍